framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/deployment/OpenShiftDeployment.java

```diff
@@ -255,34 +256,20 @@ public void setResources(Map<String, String> requests, Map<String, String> limit
                  .done();
     }
 
+    protected Optional<String> getLdapRoute(String serviceName) {
```

Comment:
Where does this route point to?

Reply:
The route will be something like ldap://ldap-${APPLICATION_NAME}.project.openshiftdomain:30389

This is according to the route name I configured in the ldap.yaml template.

Reply:
But AFAIK the route in template is available at http://ldap-${APPLICATION_NAME}.project.openshiftdomain:80
As you specify there just hostname, protocol and port are added implicitly.

Reply:
This has been reverted out.